tools/dv/samples/kmdf/pool_leak/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// License: MIT OR Apache-2.0
+
+//! # Abstract
+//!
+//!     This KMDF sample contains an intentional error that is designed to
```

Comment:
These extra spaces at the start of lines cause the doc to render incorrectly.  Please remove them.

```suggestion
//! This KMDF sample contains an intentional error that is designed to
```